Extension/package.nls.json

```diff
@@ -22,49 +22,49 @@
     "c_cpp.command.GoToNextDirectiveInGroup.title": "Go to next preprocessor directive in conditional group",
     "c_cpp.command.GoToPrevDirectiveInGroup.title": "Go to previous preprocessor directive in conditional group",
     "c_cpp.configuration.formatting.description": "Configures the formatting engine",
-    "c_cpp.configuration.formatting.clangFormat.description": "clang-format will be used to format code.",
-    "c_cpp.configuration.formatting.vcFormat.description": "The Visual C++ formatting engine will be used to format code.",
-    "c_cpp.configuration.formatting.Default.description": "By default, clang-format will be used to format the code. However, the Visual C++ formatting engine will be used if an .editorconfig file with relevant settings is found nearer to the code being formatted and clang_format_style is the default value: 'file'.",
-    "c_cpp.configuration.formatting.Disabled.description": "Code formatting will be disabled.",
-    "c_cpp.configuration.vcFormat.indent.braces.description": "Braces are indented by the amount specified in the Editor: Tab Size setting.",
-    "c_cpp.configuration.vcFormat.indent.multiLineRelativeTo.description": "Determines what new line indentation is relative to",
+    "c_cpp.configuration.formatting.clangFormat.markdownDescription": "`clang-format` will be used to format code.",
```

Comment:
Do you want to add hints for these, indicated that they may contain markdown?  I think the issue blocking proper processing of the new syntax has been addressed, though it may requiring updating the minimum required version of VS Code.

Reply:
I'd prefer to get the translators to understand the 1 rule "don't translate text inside `` unless otherwise hinted at" instead of doing things the other way around.

Reply:
I just don't think we can count on that.  I had a thread a while back with Cristiano about potentially issues with markdown contents in strings.  The conclusion seemed to be that translators are not sufficiently technical with regards to recognizing markdown and knowing the avoid breaking it.  Even with hints, it may be an issue.  :/

Reply:
But I think that requires VS 1.60.0 for package.nls.json strings, which was just recently released.

Reply:
Updating the minimum required version to 1.60.0 for our next insiders seems OK to me.  It looks like it was released last week.  I'm just trying to minimize the future work required when we inevitably get corrupted markdown from translators.  :/

Reply:
Okay, sounds the like the loc team wants a lot of hints.